src/cmap/conn/mod.rs

```diff
@@ -75,6 +75,8 @@ pub(crate) struct Connection {
 
     stream: AsyncStream,
 
+    pinned: bool,
```

Comment:
can we add a docstring to this field? ditto for the `pin` method and `unpin` method. For `unpin`, can we also mention that the connection pool is in charge of calling that method?

Reply:
Done.